SPECS-EXTENDED/aspell-en/aspell-en.spec

```
@@ -29,14 +28,16 @@ English, British English
 make
 
 %install
-rm -rf $RPM_BUILD_ROOT
-make install DESTDIR=$RPM_BUILD_ROOT
+make install DESTDIR=%{buildroot}
 
 %files
 %doc Copyright
 %{_libdir}/aspell-0.60/*
 
 %changelog
+* Mon Jun 13 2022 Jamie Magee <jamagee@microsoft.com> - 2019.10.06-5
+- Switch from FTP to HTTPS source
+
```

```suggestion
License verified
```

I checked over the license packaging and the `License` tag, and they look good.